docs/source/Plugin/P052_S11.rst

```diff
@@ -0,0 +1,141 @@
+.. include:: ../Plugin/_plugin_substitutions_p05x.repl
+.. _P052_S11_page:
+
+S11
+===
+
+|P052_typename|
+|P052_status|
+
+.. image:: P052_S11_1.jpg
+
+Introduction
+------------
+
+This is the flagship CO2 sensor from Swedish company Senseair.
+
+Specifications:
+ * 400 to 5000 ppm
+ * 5V (3.05-5.5VDC)
+ * Peak 125mA, avarage 38μA
+ * 11.5 x 33.5 x 19.7 mm
+ * UART/I2C communication
+ * More than 15 years life span
+ * No burn-in needed, will stabilize after 8 days at worst depending on placement environment
+
+Wiring
+------
+
+Circuit wiring (S11 seen from top, with the 5 vias left and the 4 vias right):
+
+.. code-block:: html
+
+    ESP               S11 (UART)
+    GPIO (14)  <-->   TX (5th via of the 5 vias)
+    GPIO (12)  <-->   RX (4th via [below TX])
+
+    Power
+    5.0V       <-->   VBB (2nd via of the 5 vias, 3.3V can also be used)
+    GND        <-->   GND (1st via)
+
+As you can see only pins on the "5 via side" are being used. But...
+
+Set up the S11 according to this simple schematics. For continuous use you should connect
+the EN (S11) to VBB (S11), EN is used to wake the sensor on demand. By using VBB this will
+make the EN go high = sensor is activated.
+
+The COMSEL pin is used if you want to use I2C communication, in that case you pull this
+low (0.4V or less). Leave open/floating if UART is used.
+
+VDDIO is supply voltage for TX/SCL/nRDY.
+
+nRDY is an output signaling when the measurement is ready. The plugin does not use that pin.
+
+DVCC is an output, only to be used for driving small loads like a level converter.
+
+.. danger::
+  Power needed: 3.05-5.5V. PLEASE OBSERVE THAT FEEDING THE S11 WITH 5.5+V WILL DAMAGE THE INTERNAL LAMP AND BREAK THE UNIT - GIVING FALSE VALUES!
+
+Setup
+-----
+
+.. image:: P052_Setup_S11_1.png
+
+Task settings
+~~~~~~~~~~~~~
+
+* **Device**: Name of plugin
+* **Name**: Name of the task (example name **CO2**)
+* **Enable**: Should the task be enabled or not
+
+Sensor
+^^^^^^
+
+* **GPIO <-- TX**: TX is generally set to **GPIO 13 (D7)**.
+* **GPIO --> RX**: RX is generally set to **GPIO 15 (D8)**.
```

Comment:
In the texts above you mentioned GPIO 12/14
GPIO 13/15 is normally used for Serial0 swapped.
There is no need to use HW serial for this sensor.

Reply:
Got it, thanks for noticing :)